Library/Homebrew/rubocops/extend/formula.rb

```diff
@@ -495,7 +495,11 @@ def string_content(node)
         when :str
           node.str_content
         when :dstr
-          node.each_child_node(:str).map(&:str_content).join
+          content = ""
+          node.each_child_node(:str, :begin) do |child|
+            content += child.type == :begin ? child.source : child.str_content
```

Comment:
```suggestion
            content += child.type == :begin ? child.source : child.str_content
```

Can you refactor the ternary operator usage into an `if`? I think it'll make this a little easier to follow.